src/lab15.py

```diff
@@ -1234,25 +1222,23 @@ def load(self, url, body=None):
             self.rules.extend(CSSParser(body.decode("utf8")).parse())
 
         images = [node
-                   for node in tree_to_list(self.nodes, [])
-                   if isinstance(node, Element)
-                   and node.tag == "img"
-                   and "src" in node.attributes]
+            for node in tree_to_list(self.nodes, [])
+            if isinstance(node, Element)
+            and node.tag == "img"
+            and "src" in node.attributes]
```

Comment:
I thought we decided to always show the broken image icon, even if `src` was specified?

Reply:
Oh—my bad. Fine, always show the `src`. When you modify the exercise, can you have me review that PR?

Reply:
ok